tezos/api/src/ffi.rs

```diff
@@ -751,6 +752,58 @@ pub struct RpcRequest {
     pub accept: Option<String>,
 }
 
+fn ffi_routed_subpath(path: &str) -> String {
+    let base = Url::parse("http://tezedge.com")
```

Comment:
@tizoc 
could we get better rid of that expect/unwraps here?

Reply:
@bkontur yes, I can just return the original and fail silently here, but if this call fails for any reason something has gone *terribly* wrong 👀